README.md

```diff
@@ -1,5 +1,47 @@
+![Netlify examples](netlify-badge-examples.png)
+
 # Netlify Feature Tour
 
-> **NOTE:** This is a work in progress.
+**Access this demo site**: https://feature-tour.netlify.app
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/fad6792e-1c44-44db-bd79-ea74b42b0f89/deploy-status)](https://app.netlify.com/sites/feature-tour/deploys)
+
+## About this example site
+
+This site provides a path to get started learning about Netlify features.
+
+- 📚 [Docs Getting Started Tutorial](https://docs.netlify.com/get-started/?utm_medium=social&utm_source=github&utm_campaign=devex-ph&utm_content=devex-examples)
+
+## Speedily deploy your own version
+
+Deploy your own version of this example site by selecting the Deploy to Netlify Button below. This will automatically:
+
+- Clone a copy of this repo to your own GitHub account
+- Create a new project in your [Netlify account](https://app.netlify.com/?utm_medium=social&utm_source=github&utm_campaign=devex&utm_content=devex-examples), linked to your new repo
+- Create an automated deployment pipeline to watch for changes on your repo
+- Build and deploy your new site
+
+[![Deploy To Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify/netlify-feature-tour&utm_medium=social&utm_source=github&utm_campaign=devex&utm_content=devex-examples)
+
+## Install and run this example locally
+
+You can clone this example repo to explore its features and implementation and to run it locally.
+
+```shell
+
+# 1. Clone the repository to your local development environment
+git clone git@github.com:netlify/feature-tour.git
+
+# 2. Move into the project directory
+cd feature-tour
```

Comment:
some unsolicited feedback noticed while reviewing the corresponding docs PR

[sand] - While these commands do work thanks to automatic redirects from GitHub, it could be nice to update them to match the updated repo name

```suggestion
# 1. Clone the repository to your local development environment
git clone git@github.com:netlify/netlify-feature-tour.git

# 2. Move into the project directory
cd netlify-feature-tour
```